src/components/MovieList/MovieCard.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { useEffect, useState } from "react";
+import Cards from "./Cards";
+import PNF from "./PNF";
+import { CardDivStyle } from "../Styles/Card.style";
+import { apiUrl, searchUrl } from "../ constants/global";
+
+const MovieCard = ({ input }) => {
+  const [data, setData] = useState([]);
+
+  const getMovie = async (url) => {
+    const res = await fetch(url);
+    const resResults = await res.json();
```

Comment:
Please add error handling. Always wrap the API calls inside try catch block and handle errors if any.